src/components/Footer/Footer.js

```diff
@@ -67,11 +85,84 @@ export default class Footer extends Component {
     }, 1000);
   }
 
+  renderFooterHeader() {
+    const logoLink = "https://assets.ctl.io/";
+    return (
+      <div className={FOOTER_CLASSES.FOOTER_HEADER}>
+        <Brand 
+          size="md (20px)"
+          color="default"
+        />
+        <div className={FOOTER_CLASSES.FOOTER_SEARCH}>
+          <div className={FORM_CLASSES.ITEM}>
+            <label className={`${LABEL_CLASSES.LABEL} ${UTILITY_CLASSES.DISPLAY.NONE}`} htmlFor={FOOTER_CLASSES.FOOTER_SEARCH_INPUT}>
+              Search Lumen
+            </label>
```

Comment:
can we use web components or exisitng uxpin components here? The same for all places where we have inputs, a...

Reply:
@jllr I have refactored the code to update it with web components wherever needed.